nmrfx-core/src/main/java/org/nmrfx/chemistry/relax/OrderPar.java

```diff
@@ -398,24 +421,63 @@ public String toString() {
         RelaxationValues.appendValueError(sBuilder, TauS, TauSerr,"%.4f");
         RelaxationValues.appendValueError(sBuilder, Rex, Rexerr,"%.2f");
         sBuilder.append("\t").append(model).append("\t").append(modelNum);
+        sBuilder.append("\t").append(String.format("%.4f", sumSqErr)).append("\t").
+                append(String.format("%.4f", getReducedChiSqr())).
+                append("\t").append(String.format("%.4f",getAIC())).append("\t").
+        append(nValues).append("\t").append(nPars);
```

Comment:
This is more of a style thing, but I find it easier to read when the dots are on the start of a row
```suggestion
        sBuilder.append("\t").append(String.format("%.4f", sumSqErr)).append("\t")
                .append(String.format("%.4f", getReducedChiSqr()))
                .append("\t").append(String.format("%.4f",getAIC())).append("\t")
                .append(nValues).append("\t").append(nPars);
```

Reply:
That is a style thing and I don't know if there is a standard for this.  I like the dots at the end because when I read the line to the end and see a dot I know that it is continued on the next line.

Reply:
I've cleaned up some of the sonar hints